docs/content/architecture/pgadmin4.md

```diff
@@ -55,7 +55,8 @@ PGPASSWORD=$(kubectl get secrets -n postgres-operator "${PG_CLUSTER_USER_SECRET_
 PGUSER=$(kubectl get secrets -n postgres-operator "${PG_CLUSTER_USER_SECRET_NAME}" -o go-template='{{.data.user | base64decode}}')
 ```
 
-Though the prompt says "email address", using your PostgreSQL username will work.
+To login you will need to add the `@pgo` suffix to your username. For example, in the image below
+the username is `hippo` and we update it to `hippo@pgo`.
```

Comment:
⛏️ I'm not sure we want to mention the image; not everyone will be able to see it. Perhaps the suffix can be explained in the paragraph above this and the one here can go away?

I'm not quite sure how to phrase it all...

> Point your browser at `http://localhost:5050` and you will be prompted to log in.
> Use your database username with `@pgo` appended and your database password.
> In our case, the pgAdmin username is `hippo@pgo` and the password is found in the
> user secret, `hippo-pguser-hippo`: